code/obj/artifacts/artifact_items/energy_gun.dm

```diff
@@ -8,6 +8,7 @@
 	is_syndicate = 1
 	mat_changename = 0
 	mat_changedesc = 0
+	var/can_swap_cell = 0
```

Comment:
```suggestion
	can_swap_cell = 0
```

Reply:
Oh yeah turns out doing a lazy copy and paste and not double checking is a bad.